jira_agile_metrics/config_test.py

```diff
@@ -40,9 +40,8 @@ def test_config_to_options_minimal():
 
     assert options['settings']['backlog_column'] == 'Backlog'
     assert options['settings']['committed_column'] == 'In progress'
-    assert options['settings']['final_column'] == 'In progress'
```

Comment:
And again - changes a test that would have passed previously.

Reply:
`final_column` is not used anymore, therefore I would find it confusing to have a test for it.